### PR TITLE
HAWQ-1151 - add ambari procedures

### PR DESCRIPTION
**Pull request by lisakowen**
update docs to identify specific ambari procedures when modifying server configuration parameters.

### CODE REVIEW

File: ddl/ddl-table.html.md.erb [hunk 1]
```diff
@@ -93,14 +93,14 @@ For any specific query, the first four factors are fixed values, while the confi
...
 
-**Note:** For best performance with large tables, the number of buckets should not exceed the value of the `default_hash_table_bucket_number` parameter. Small tables can use one segment node, `with bucketnum=1`. For larger tables, the bucketnum is set to a multiple of the number of segment nodes, for the best load balancing on different segment nodes. The elastic runtime will attempt to find the optimal number of buckets for the number of nodes being processed. Larger tables need more virtual segments , and hence use larger numbers of buckets.
+**Note:** For best performance with large tables, the number of buckets should not exceed the value of the `default_hash_table_bucket_number` parameter. Small tables can use one segment node, `WITH bucketnum=1`. For larger tables, the `bucketnum` is set to a multiple of the number of segment nodes, for the best load balancing on different segment nodes. The elastic runtime will attempt to find the optimal number of buckets for the number of nodes being processed. Larger tables need more virtual segments , and hence use larger numbers of buckets.
```
Comment (dyozie) [inline ddl/ddl-table.html.md.erb]: Might as well fix (remove) the space before the comma here.

File: ddl/ddl-table.html.md.erb [hunk 2]
```diff
@@ -132,7 +132,7 @@ Policies for different application scenarios can be specified to optimize perfor
...
+Hash tables are created using the value specified in `default_hash_table_bucket_number`. Explicitly setting this value can be useful in managing resources, as queries for hash tables use a fixed number of buckets, regardless of the amount of data present. If a larger or smaller number of hash buckets are desired, set this value before you CREATE TABLE. Resources are dynamically allocated to a multiple of the number of nodes. If using `hawq init --bucket_number` to set the value of `default_hash_table_bucket_number` during cluster initialization or expansion, the value should not exceed the value of `hawq_rm_nvseg_perquery_limit`, which defines the maximum number of virtual segments that can be used for a query \(default = 512, with a maximum of 65535\). Modifying the value to greater than 1000 segments is not recommended.
```
Comment (dyozie) [inline ddl/ddl-table.html.md.erb]: change "CREATE TABLE" to "create tables"  Also might want to tackle some of the passive voice in this paragraph.

File: plext/using_pljava.html.md.erb [hunk 3]
```diff
@@ -77,6 +77,41 @@ If TRUE, lingering savepoints will be released on function exit. If FALSE, they
 
 Defines the start up options for the Java VM.
 
+### <a id="setting_serverconfigparams"></a>Setting PL/Java Configuration Parameters 
+
+You can set PL/Java server configuration parameters at the session level, or globally across your whole cluster. Your HAWQ cluster configuration must be reloaded after setting a server configuration value globally.
```
Comment (dyozie) [inline plext/using_pljava.html.md.erb]: See previous edits.

File: plext/using_pljava.html.md.erb [hunk 4]
```diff
@@ -179,15 +203,25 @@ Replace \<dbname\> with the name of the target database.
 
 When no databases have PL/Java as a registered language, remove the Java JAR files:
 
-1. Remove the `pljava_classpath` server configuration parameter in the `hawq-site.xml` file.
+1. Remove the `pljava_classpath` server configuration parameter:
```
Comment (dyozie) [inline plext/using_pljava.html.md.erb]: Better to make following procedure two separate procs, one for ambari and one cli.

File: query/gporca/query-gporca-enable.html.md.erb [hunk 5]
```diff
@@ -20,35 +20,45 @@ As part of routine database maintenance, you should refresh statistics on the ro
 
 When the configuration parameter `optimizer_analyze_root_partition` is set to `on`, root partition statistics will be collected when `ANALYZE` is run on a partitioned table. Root partition statistics are required by GPORCA.
 
-1.  Log into the HAWQ master host as `gpadmin`, the HAWQ administrator.
-2.  Set the values of the server configuration parameters. These HAWQ `hawq             config` utility commands sets the value of the parameters to `on`:
+You will perform different procedures to set optimizer configuration parameters for your whole HAWQ cluster depending upon whether you manage your cluster from the command line or use Ambari. If you use Ambari to manage your HAWQ cluster, you must ensure that you update server configuration parameters via the Ambari Web UI only. If you manage your HAWQ cluster from the command line, you will use the `hawq config` command line utility to set optimizer server configuration parameters.
```
Comment (dyozie) [inline query/gporca/query-gporca-enable.html.md.erb]: See previous edits.

**Comment by lisakowen** [comment]
thanks for reviewing, david.  i believe i have incorporated all of your changes.

**Comment by dyozie** [comment]
Thanks, Lisa!  Can you squash these commits before I go ahead and merge the PR?

**Comment by lisakowen** [comment]
this has been merged.

**Pull request closed without merging**